zoo-css-challenge/index.html

```diff
@@ -1,136 +1,233 @@
+          submits and walks away.
+        </p>
+
+        <h3>Height and Size</h3>
+
+        <p>
+          Giraffes use their height to good advantage and browse on leaves and
+          buds in treetops that few other animals can reach (acacias are a
+          favorite). Even the giraffe's tongue is long! The 21-inch tongue helps
+          them pluck tasty morsels from branches. Giraffes eat most of the time
+          and, like cows, regurgitate food and chew it as cud. A giraffe eats
+          hundreds of pounds of leaves each week and must travel miles to find
+          enough food.
+        </p>
+      </section>
+
+      <!-- 10) New Section: Added new section on Labradors! -->
+      <section class="labs">
+        <h2>Labrador</h2>
+        <p>
+          Labrador's are the most popular dog breed in the UK and can be one of
+          three colours.
+        </p>
+        <div class="image-container-lab">
+          <img
+            src="images/Black_Labrador.jpg"
+            alt="Black Labrador"
+            width="400"
+            height="400"
+          />
+          <img
+            src="images/Golden_Labrador.jpg"
+            alt="Golden Labrador"
+            width="200"
+            height="400"
+          />
+          <img
+            src="images/Chocolate_Labrador.jpg"
+            alt="Chocolate_Labrador"
+            width="200"
+            height="400"
+          />
+        </div>
+      </section>
+
+      <section class="news">
+        <h2>News</h2>
+        <!-- 5) News: Added a 3rd news item, ordered by date -->
+        <div class="card">
+          <time>2021-05-30</time>
+          <h4>Which big cat are you?</h4>
+          <a>Find out More</a>
+        </div>
+        <div class="card">
+          <time>2010-04-16</time>
+          <h4>Helping wild turtles in Seychelles</h4>
+          <a>Find out More</a>
+        </div>
+        <div class="card">
+          <time>2010-04-12</time>
+          <h4>Learning about the rainforest</h4>
+          <a>Find out More</a>
+        </div>
+      </section>
+
+      <section class="education">
+        <div class="education-container">
+          <h3>Learning resources for schools</h3>
+          <p>
+            Explore our activities, videos, and lesson plans to support a wide
+            range of curriculum topics
+          </p>
+          <button>Find them here</button>
+        </div>
+      </section>
+
+      <section class="membership">
+        <h2>Are there any benefits or discounts for members?</h2>
+        <p>
+          There are <strong>AMAZING</strong> benefits of becoming a member with
+          Birmingham Zoo! Our members get special discounts in our shops, cafes,
+          and unlimited entry
+        </p>
+        <button>Learn More</button>
+      </section>
+
+      <section class="programs">
+        <h2>Programs</h2>
+
+        <div class="progCardContainer">
```

Comment:
```suggestion
        <div class="prog-card-container">
```
We should keep the case of our classes the same. We use kebab case in another class `image-container-lab` so we can use it here too for `prog-card-container`

Reply:
I wondered about the correct convention for classes as camel case and pascal case are used for general programming I went with camel but have never been sure about classes, thanks for the feedback!

Reply:
As a follow up, which case should ID's generally follow?

Reply:
Typically, HTML and CSS uses kebab-case, so that means kebab-case for HTML attributes, CSS class names and IDs. Again, some software teams might decide on another case, but the most important thing in any codebase is that it is consistent so if we use kebab-case for a CSS class name then we should use kebab-case for all CSS class names